lib/Widget/dashboard/dashboard_Table.dart

```diff
@@ -201,252 +202,260 @@ class _DashboardTableState extends State<DashboardTable> {
+                              ),
+                              for (var item in dashboardTable)
+                                Container(
+                                  color: Color(0xFFC37447),
+                                  child: SizedBox(
+                                    child: Column(children: [
+                                      Text(
+                                        item['Date'],
+                                        style: TextStyle(
+                                            fontStyle: FontStyle.italic,
+                                            color: Colors.white,
+                                            fontSize: 15),
+                                      ),
+                                      Text(
+                                        item['Time'],
+                                        style: TextStyle(
+                                            fontStyle: FontStyle.italic,
+                                            color: Colors.white,
+                                            fontSize: 15),
+                                      )
+                                    ]),
+                                  ),
+                                ),
+                            ]),
+                            TableRow(children: [
+                              TableCell(
+                                verticalAlignment:
+                                    TableCellVerticalAlignment.fill,
+                                child: Container(
+                                  color: Colors.grey[100],
                                   child: Column(children: [
-                                    Text(
-                                      'Date',
-                                      style: TextStyle(
-                                          fontStyle: FontStyle.italic,
-                                          color: Colors.white,
-                                          fontSize: 16),
-                                    ),
+                                    Text(' Days after Admission ',
+                                        style: TextStyle(
+                                            fontStyle: FontStyle.italic,
+                                            fontSize: 15))
                                   ]),
                                 ),
                               ),
-                            ),
-                            for (var item in dashboardTable)
-                              Container(
-                                color: Color(0xFFC37447),
-                                child: SizedBox(
+                              for (var item in dashboardTable)
+                                Column(children: [
+                                  Text(item['Admission'].toString(),
+                                      style: TextStyle(fontSize: 16))
+                                ]),
+                            ]),
```

Comment:
Just wonder. Why we need to for loop every single MapKey on the same map.

Why cant' we Loop once. This one looks like if the map has 10 key with 10elements. it's gonna be 10x10 time loop

Reply:
@PormpimonPang

Reply:
<img width="1183" alt="Screen Shot 2564-05-09 at 13 55 19" src="https://user-images.githubusercontent.com/57941667/117563210-82d9de80-b0ce-11eb-8705-04f334cae330.png">
@Thaweesak-sai  When it prints, it will print row by row. The table header is on the first column and can't config the first column as the table header, so I have to loop for print every row.